packages/ui-kit/src/components/Logger/index.ts

```diff
@@ -0,0 +1,2 @@
+export { useLog } from './useLog'
+export * from './LogProvider'
```

Comment:
We should probably only export LogProvider and LogProviderProps, to avoid polluting the public API?

Also, we need to export the Log and LogLevel types, in addition to `useLog`. Something like this?

```ts
export { Log, LogLevel } from './Log.types'
export { LogProvider, LogProviderProps } from './LogProvider'
export { useLog } from './useLog'
```